src/triage/component/results_schema/__init__.py

```diff
@@ -88,6 +90,47 @@ def stamp_db(revision, dburl):
     command.stamp(alembic_config(dburl=dburl), revision)
 
 
+def upgrade_if_clean(dburl):
+    """Upgrade the database only if the results schema hasn't been created yet.
+
+    Raises: ValueError if the database results schema version does not equal the code's version
+    """
+    alembic_cfg = alembic_config(dburl)
+    engine = create_engine(dburl)
+    script_ = script.ScriptDirectory.from_config(alembic_cfg)
+    if not table_exists('results_schema_versions', engine):
+        logging.info("No results_schema_versions table exists, which means that this installation"
+                     "is fresh. Upgrading db.")
+        upgrade_db(dburl=dburl)
+        return
+    with engine.begin() as conn:
+        current_revision = list(conn.execute('select version_num from results_schema_versions limit 1'))[0][0]
+        logging.info("Database's results schema version is %s", current_revision)
+        triage_head = script_.get_current_head()
+        logging.info("Code's results schema version is %s", triage_head)
+        database_is_ahead = current_revision not in set(migration.revision for migration in script_.walk_revisions())
+        if database_is_ahead:
+            raise ValueError(
+                "Your database's results schema version, %s, is not a known revision to this"
+                "version of Triage knows about. Usually, this happens if you use a branch with a"
+                "new results schema version and upgrade the database to that version. To use this"
+                "version of Triage, you will likely need to check out that branch out"
+                "and downgrade to %s",
+                current_revision,
+                triage_head
+            )
+        elif current_revision != triage_head:
+            raise ValueError(
+                "Your database's results schema revision, %s, does not equal the revision used"
+                "by this version of Triage. However, your database can be upgraded to this "
+                "revision. If you would like to upgrade the database, call "
```

Comment:
just for clarity, rather than "call", perhaps stress that we're talking about the CLI, _e.g._ 🤷‍♂ 

> If you would like to upgrade your database from the console, and you've installed Triage, you may execute: `triage db upgrade`. If the `triage` command is unavailable, (because you are running Triage directly from a repository checkout), then: `manage alembic upgrade head`.